activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/rest/EnableActivitiRestQueryServiceTest.java

```diff
@@ -1,181 +0,0 @@
-/*
- * Copyright 2018 Alfresco, Inc. and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.activiti.cloud.services.query.rest;
-
-import static org.hamcrest.CoreMatchers.endsWith;
-import static org.hamcrest.CoreMatchers.hasItems;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
-import org.activiti.cloud.services.test.rest.support.TestMvcClient;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
-import org.springframework.context.annotation.Bean;
-import org.springframework.data.rest.webmvc.RestMediaTypes;
-import org.springframework.hateoas.Link;
-import org.springframework.hateoas.LinkDiscoverer;
-import org.springframework.hateoas.LinkDiscoverers;
-import org.springframework.hateoas.core.JsonPathLinkDiscoverer;
-import org.springframework.http.MediaType;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-import org.springframework.web.context.WebApplicationContext;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest(
-        webEnvironment = WebEnvironment.MOCK,
-        properties = {"spring.jpa.hibernate.ddl-auto=create-drop", "spring.jpa.show-sql=true"
-        })
-public class EnableActivitiRestQueryServiceTest {

```

Comment:
I'm not sure that we want to remove those tests. Were they depending on spring data rest? Can't we have these tests running using controllers?

Reply:
We removed the annotation EnableActivitiRestQueryService.  When we have test controllers we don't need the vast majority of those tests here just checking the presence of the controllers. Also some tests should explain what they do, there was one that honestly was not clear to me using RestMediaTypes.ALPS_JSON. Why?

Reply:
Also there was a test for a repository that was not testing anything :)